ramda.js

```diff
@@ -4705,6 +4707,42 @@
     R.gte = op(function _gte(a, b) { return a >= b; });
 
 
+    function createMaxMin(comparator, initialVal) {
+        return function(list) {
+            if (arguments.length === 0) {
+                throw noArgsException();
+            }
+            var idx = -1, winner = initialVal, computed;
+            while (++idx < list.length) {
+                computed = +list[idx];
+                if (comparator(computed, winner)) {
+                    winner = computed;
+                }
+            }
+            return winner;
+        };
+    }
+
+    function createMaxMinWith(comparator) {
+        return function(valueComputer, list) {
+            if (!(list && list.length > 0)) {
+                return;
```

Comment:
> Your description for max/min sound more like

I not describing `max/min` here I'm talking about `maxBy`/`minBy`.
I just think it odd that `maxBy/minBy` have entirely different behavior than `min/max`.

I would expect passing `maxBy/minBy` and identity function to have the same output as `max/min`.
Maybe that's complicated here because `max/min` return coerced values, I donno.